subworkflows/local/standardisation_profiles.nf

```diff
@@ -33,7 +33,7 @@ workflow STANDARDISATION_PROFILES {
                             .groupTuple ()
                             .map { [ it[0], it[1].flatten() ] }
 
-    TAXPASTA_MERGE (ch_input_for_taxpasta, [], [])
+    TAXPASTA_MERGE (ch_input_for_taxpasta, params.taxpasta_taxonomy_dir ? params.taxpasta_taxonomy_dir : [], [])
```

Comment:
Does this need a `Channel.fromPath(params.taxpasta_taxonomy_dir, checkIfExists: true)` or something like that to check for the presence of that directory early on?

Reply:
Ah yes!

Reply:
I think this would also be good to stage the files correctly into the module

Reply:
Yup, it's incoming - waiting for local test to finish :)